draft-irtf-cfrg-opaque.md

```diff
@@ -1016,6 +997,24 @@ the identity is acceptable or not to the peer. However, we note that the
 public keys of both the server and the user must always be those defined at
 time of password registration.
 
+# OPAQUE Configurations {#configurations}
+
+An OPAQUE configuration must specify the OPRF protocol variant and MHF function.
+OPAQUE uses the OPRF protocol from {{I-D.irtf-cfrg-voprf}}, and supports the following
+ciphersuites:
+
+- OPRF-curve25519_XMD:SHA-512_ELL2_RO_
+- OPRF-curve448_XMD:SHA-512_ELL2_RO_
+- OPRF-P256_XMD:SHA-256_SSWU_RO_
+- OPRF-P384_XMD:SHA-512_SSWU_RO_
+- OPRF-P521_XMD:SHA-512_SSWU_RO_
+
+Supported MHFs include Argon2 {{?I-D.irtf-cfrg-argon2}}, scrypt {{?RFC7914}},
+and PBKDF2 {{?RFC2898}} with suitable parameter choices. These may be constant
+values or set at the time of password registration and stored at the server.
+In the latter case, the server communicates these parameters to the client during
+login.
```

Comment:
It is TBD how the server does that

Reply:
Yeah, we'll need to work out a way to do this before revving the draft.